spec/karma.conf.js

```diff
@@ -1,44 +1,22 @@
-// Karma configuration
 module.exports = function (config) {
-
-    var libSources = [
-        'dist/js/L.DNC.js'
-    ];
-
-    var testData = [
-        'spec/testing-data.js'
-    ];
-
-    var leafletSources = require(__dirname+'/../node_modules/leaflet/build/build.js').getFiles();
-
-    for (var i=0; i < leafletSources.length; i++) {
-        leafletSources[i] = __dirname+"/../node_modules/leaflet/" + leafletSources[i];
-    }
-
-    var files = [
-        "spec/es5-shims.js",
-        "spec/sinon.js",
-        "spec/expect.js"
-    ].concat(leafletSources, testData, libSources, [
-        "node_modules/happen/happen.js",
-        "spec/suites/SpecHelper.js",
-        "spec/suites/**/*.js",
-    ]);
+    'use strict';
 
     config.set({
+
         // base path, that will be used to resolve files and exclude
         basePath: '../',
 
-        plugins: [
-            'karma-mocha',
-            'karma-phantomjs-launcher',
-            'karma-chrome-launcher'],
-
         // frameworks to use
-        frameworks: ['mocha'],
+        frameworks: ['mocha', 'chai', 'sinon'],
 
         // list of files / patterns to load in the browser
-        files: files,
+        files: [
+            "node_modules/leaflet/dist/leaflet.js",
```

Comment:
I feel like this should be `mapbox.js` as that's what we're actually using in prod.  Differences seem to be minimal, but would prefer to keep tests as close to production as possible.